dashboard/state_aggregator.py

```diff
@@ -648,7 +648,9 @@ async def summarize_tasks(self, option: SummaryApiOptions) -> SummaryApiResponse
         # For summary, try getting as many entries as possible to minimze data loss.
         result = await self.list_tasks(
             option=ListApiOptions(
-                timeout=option.timeout, limit=RAY_MAX_LIMIT_FROM_API_SERVER, filters=[]
+                timeout=option.timeout,
+                limit=RAY_MAX_LIMIT_FROM_API_SERVER,
+                filters=option.filters,
             )
         )
         summary = StateSummary(
```

Comment:
@rkooo567 @rickyyx , should I update the SummaryApiResponse `num_filtered` value?

It's not clear what this is supposed to be.. Should it show the number of results from the summary api (1 per task name after filtering) or the number of tasks after filtering?

Reply:
I think we can just use the same output that's returned from list_tasks?